x-pack/solutions/search/plugins/enterprise_search/server/index.ts

```diff
@@ -44,7 +44,16 @@ export const configSchema = schema.object({
 export type ConfigType = TypeOf<typeof configSchema>;
 
 export const config: PluginConfigDescriptor<ConfigType> = {
-  deprecations: ({ unused }) => [unused('canDeployEntSearch', { level: 'warning' })],
+  deprecations: ({ deprecate, unused }) => [
+    unused('canDeployEntSearch', { level: 'warning' }),
+    deprecate('host', '9.0.0', { level: 'critical' }),
```

Comment:
I think the only question that I have is the second parameter here, the _minimum_ version number? Or is it an absolute version?